pkg/steps/source_test.go

```diff
@@ -685,7 +685,7 @@ RUN rm -f /oauth-token
 
 	for _, testCase := range testCases {
 		t.Run(testCase.name, func(t *testing.T) {
-			if actual, expected := createBuild(testCase.config, testCase.jobSpec, testCase.clonerefsRef, testCase.resources, testCase.cloneAuthConfig), testCase.expected; !equality.Semantic.DeepEqual(actual, expected) {
+			if actual, expected := createBuild(testCase.config, testCase.jobSpec, testCase.clonerefsRef, testCase.resources, testCase.cloneAuthConfig, nil), testCase.expected; !equality.Semantic.DeepEqual(actual, expected) {
```

Comment:
It's better to have a test case for this.

Reply:
Will work on this.

Reply:
Test case updated.